src/sentry/integrations/vsts/integration.py

```diff
@@ -221,8 +221,15 @@ def build_integration(self, state):
 
     def create_subscription(self, instance, account_id, oauth_data):
         webhook = WorkItemWebhook()
-        subscription, shared_secret = webhook.create_subscription(
-            instance, oauth_data, self.oauth_redirect_url, account_id)
+        try:
+            subscription, shared_secret = webhook.create_subscription(
+                instance, oauth_data, self.oauth_redirect_url, account_id)
+        except ApiError as e:
+            if 'does not have permission to edit a subscription' in e.message:
```

Comment:
Does the response not have an appropriate status code? Theoretically should be a 403.